TribalWars/src/ApplicationCore/Entities/RequiredBuilding.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ApplicationCore.Entities
+{
+    public class RequiredBuilding
+    {
+        public int Id { get; set; }
+        public int ReqBuildingId { get; set; }
+        public int Level { get; set; }
+        public virtual ICollection<BuildingRequiredBuilding> Buildings { get; set; }
+        public virtual Building ReqBuilding { get; set; }
```

Comment:
Unnecessary shortcut in ReqBuilding and name of collection Buildings does not apply to Building entity.

Reply:
The shortcut is because I cannot name it RequiredBuilding

Reply:
Can we think of another name that describes this property intention? I'm not a fan of this shortcut too.

Reply:
Maybe just Building and BuildingId?